test/index.spec.js

```diff
@@ -1,9 +1,11 @@
+/* eslint max-nested-callbacks: ["error", 8] */
 /* eslint-env mocha */
 'use strict'
-
+require('loud-rejection')()
```

Comment:
"facepalm" that a module had to be created to enable this.

Let's put a comment on top saying what it is for